cmd/api/config.go

```diff
@@ -53,6 +53,13 @@ type HTTPConfig struct {
 	MaxRequestPerInterval uint64 `default:"10"`
 }
 
+// WebhookConfig contains configuration for the Table events Webhook.
+type WebhookConfig struct {
```

Comment:
New webhook config is introduced at the `EventProcessor` level.

Reply:
The first two points make sense. Removing the `enabled` flag and adding a check to see if the URL is not an empty string. Still in doubt about the third point. Please see the other comment.

Reply:
> Still in doubt about the third point. Please see the other comment.

It's not strictly needed. As you said, we could parse the URL to decide which implementation to use. I'm fine either way.